src/common/selftest/selftest_MINI.cpp

```diff
@@ -278,25 +278,16 @@ void CSelftest::log_open() {
         serial_otp = 0;
     }
     char serial[32] = "unknown";
-    const char *suffix = "";
-    switch (m_Mask) {
-    case stmFans:
+    const char *suffix = "_no_suffix";
+    if (m_Mask & stmFans)
         suffix = "_fans";
-        break;
-    case stmXYAxis:
-    case stmHome_XYAxis:
+    if (m_Mask & stmXYAxis)
```

Comment:
can we have multiple suffices? This code looks like we could (no "else if"), but the suffix pointer is always changed hard

Reply:
else if will be used